packages/block-editor/src/rating-scale-answer/style.scss

```diff
@@ -0,0 +1,5 @@
+.wp-block[data-type="crowdsignal-forms/rating-scale-question"]:not(.is-selected):not(.has-child-selected) {
```

Comment:
Actually, this should go into the theme compatibility layer. If it's a default to be shared by all themes, put it in `base/editor.scss`.

Reply:
I'd ask why this is necessary?